### PR TITLE
Allow for full replay of the data folder

### PR DESCRIPTION
**Pull request by nlisgo**
If this is adopted I will update the "publish manuscript" issue template: https://github.com/elifesciences/enhanced-preprints-import/blob/master/.github/ISSUE_TEMPLATE/publish_manuscript.md

### CODE REVIEW

File: scripts/fetch_meca_archives.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+output_dir="$(realpath ${2-${PARENT_DIR}/incoming})" # default ./incoming
+manuscripts_txt="$(realpath ${3-${PARENT_DIR}/manuscripts.txt})" # default ./manuscripts.txt
+meca_lookup="$(realpath ${3-${PARENT_DIR}/meca-lookup.txt})" # default ./meca-lookup.txt
+
+echo "Retrieving ${all_or_new} manuscripts in ${manuscripts_txt}"
+
+while read -r line; do
+    if [[ "$all_or_new" = "all" || ! -d "${PARENT_DIR}/data/${line}" ]]; then
+        $SCRIPT_DIR/fetch_meca_archive.sh "$line" $output_dir $meca_lookup
```
Comment (fred-atherden) [inline scripts/fetch_meca_archives.sh:27]: Was trialling this over in https://github.com/elifesciences/enhanced-preprints-data/pull/54 with 10.21203/rs.3.rs-2200020/v3.

When I run `./scripts/fetch_meca_archives.sh`, looks like I get an error when trying to invoke `fetch_meca_archive.sh`. Looks like it's due to `readlink`s `-f` option and not being able to retrieve `$SCRIPT_DIR`. Is there a dependency (aside from `jq`) I need to install?

```
$ ./scripts/fetch_meca_archives.sh     
                                             
readlink: illegal option -- f
usage: readlink [-n] [file ...]
Retrieving new manuscripts in /Users/fredatherden/Documents/GitHub/enhanced-preprints-data/manuscripts.txt
./scripts/fetch_meca_archives.sh: line 27: ./fetch_meca_archive.sh: No such file or directory
```

Comment (fred-atherden) [reply scripts/fetch_meca_archives.sh:27]: Nevermind. Aliasing readlink to greadlink worked for me

**Review by fred-atherden — APPROVED** [review]
Tested out in https://github.com/elifesciences/enhanced-preprints-data/pull/54

File: scripts/fetch_meca_archives.sh (hunk 1 above)
Comment (nlisgo) [reply scripts/fetch_meca_archives.sh:27]: `jq`, `sed` and `xmllint` are needed. I could run through docker if you anticipate any issues.

Comment (fred-atherden) [reply scripts/fetch_meca_archives.sh:27]: Working fine for me locally after the workaround above.

**Pull request merged**